src/translations/en.json

```diff
@@ -359,6 +359,30 @@
     "LoadCopyFailed404": "Load copy failed: the load was not found",
     "LoadCopyFailed": "Load copy failed",
 
+    "LineCopied": "Data successfully copied from line {lineId}",
```

Comment:
Maybe define only following labels : 'EquipmentCopied', 'EquipmentCopyFailed404' and 'EquipmentCopyFailed' ?

Reply:
probably better to avoid adding that at each new equipment yeah, done.